ports/raspberrypi/common-hal/wifi/Radio.c

```diff
@@ -131,7 +131,8 @@ void common_hal_wifi_radio_set_mac_address_ap(wifi_radio_obj_t *self, const uint
     ro_attribute(MP_QSTR_mac_address_ap);
 }
 
-mp_obj_t common_hal_wifi_radio_start_scanning_networks(wifi_radio_obj_t *self) {
+mp_obj_t common_hal_wifi_radio_start_scanning_networks(wifi_radio_obj_t *self, uint8_t start_channel, uint8_t stop_channel) {
+    // channel bounds are ignored; not implemented in driver
```

Comment:
maybe raise not implemented error when provided channel range doesn't match the default range

Reply:
I don't actually know which channels are scanned by default. I looked for documentation, but maybe I am looking in the wrong place. These config params are set to zero:
```c
    int32_t channel_num;
    uint16_t channel_list[1];
```

@jepler Do you know, or know where to look?

Reply:
As far as I can tell there's no way to set the channels to scan in picow. some internal default is always used. It may be controlled by the country setting, which we never change.